WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -378,6 +381,47 @@ public static PostModel updatePostWithCurrentlyFailedUploads(PostModel post) {
         return post;
     }
 
+    // this method aims at recovering the current state of media items if they're inconsistent within the PostModel.
+    public static void resetUploadingMediaToFailedIfNotInProgressOrQueued(Context context, PostModel post) {
```

Comment:
Relying on Aztec status feels very out of place here - since this is only being called in `onCreate()` of the `EditPostActivity`, why not move this method into `AztecEditorFragment` (and expose any necessary `UploadService` methods so they can be called from there)?